homeassistant/components/roon/media_player.py

```diff
@@ -136,6 +136,7 @@ async def async_added_to_hass(self) -> None:
             )
         )
         self._server.add_player_id(self.entity_id, self.name)
+        self._server.add_player_volume_hook(self.entity_id, self.name)
```

Comment:
Can we create an event entity instead?

https://developers.home-assistant.io/docs/core/entity/event